newIDE/app/src/AssetStore/AssetDetails.js

```diff
@@ -129,6 +130,8 @@ export const AssetDetails = ({
     () => {
       (async () => {
         try {
+          // Reinitialise asset to trigger a loader and recalculate all parameters. (for instance zoom)
+          setAsset(null);
           const loadedAsset = await getAsset(assetShortHeader);
           setAsset(loadedAsset);
```

Comment:
Why wasn't the new asset size taken into account without it?

Reply:
The ImagePreview component initialize a state value for its zoom based on the props, but does not update this state value when the prop changes.
We can either:
- Never clear the asset (just replace it) meaning that we don't see a loader (the network call happens in the background) when a user clicks on an asset or when they click "back"
- Always clear the asset, triggering a loader that can be really short or long depending on network connection.

I feel like solution 1 could be confusing if a user has a slow connection, they wouldn't have the feedback that their asset is currently loading